roles/puppet_agent/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: manage puppet package
+  package:
+    name: "{{ puppet_package_name }}"
+    state: "{{ puppet_package_state }}"
+  when: puppet_package_manage
+
+- name: ensure puppet group
+  group:
+    name: "{{ puppet_group_name }}"
+    state: "{{ puppet_group_state }}"
+  when: puppet_group_manage
+
+- name: puppet ssl bootstrap
+  when: puppet_bootstrap_manage|bool
+  block:
+    # This makes sure 'sudo puppet' works which is needed for bootstrapping
+    - name: setup sudo secure_path
+      copy:
+        content: "Defaults secure_path = \"/sbin:/bin:/usr/sbin:/usr/bin:/opt/puppetlabs/bin\"\n"
```

Comment:
```suggestion
        content: 'Defaults secure_path = "/sbin:/bin:/usr/sbin:/usr/bin:/opt/puppetlabs/bin"\n'
```

Reply:
I tried that, but then it literally outputs a `\n` in the file and you break sudo.

Reply:
ugh. ok